go/teams/service_helper.go

```diff
@@ -154,26 +154,25 @@ func Details(ctx context.Context, g *libkb.GlobalContext, name string) (res keyb
 	if err != nil {
 		return res, err
 	}
-	return details(ctx, g, t, tracer)
+	return details(libkb.NewMetaContext(ctx, g), t, tracer)
 }
 
-func details(ctx context.Context, g *libkb.GlobalContext, t *Team, tracer profiling.TimeTracer) (res keybase1.TeamDetails, err error) {
+func details(mctx libkb.MetaContext, t *Team, tracer profiling.TimeTracer) (res keybase1.TeamDetails, err error) {
 	res.KeyGeneration = t.Generation()
 	tracer.Stage("members")
-	res.Members, err = MembersDetails(ctx, g, t)
+	res.Members, err = MembersDetails(mctx.Ctx(), mctx.G(), t)
 	if err != nil {
 		return res, err
 	}
 
 	tracer.Stage("invites")
-	annotatedInvites, err := AnnotateInvitesUIDMapper(ctx, g, t, &res.Members)
+	res.AnnotatedActiveInvites, err = AnnotateInvitesNoPUKless(mctx, t, &res.Members)
 	if err != nil {
 		return res, err
```

Comment:
Generally if err != nil there's no contract for the other values, with few exceptions (probably should be zero exceptions).

Reply:
yeah. I think it is good practice to return empty structs/nil in case of errors so that if the caller forgets to check the error it is easier to notice.